doc/users/next_whats_new/axline_transform.rst

```diff
@@ -0,0 +1,7 @@
+axline supports *transform* parameter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`~.Axes.axline` now supports the *transform* parameter, which applies to
+the points *xy1*, *xy2*.
+*xy2*. The *slope* (if given) is always in data coordinates. This can
```

Comment:
Duplicate *xy2*. Also, you can re-wrap this line.

Reply:
oops, sorry. Done in 82e63a98cd720938.